react/components/molecules/keyboard-numeric/keyboard-numeric.js

```diff
@@ -65,7 +65,8 @@ const styles = StyleSheet.create({
         marginBottom: 5
     },
     filler: {
-        flex: 1
+        flex: 1,
+        marginHorizontal: 2

```

Comment:
Can you explain where the value came from? I understand that empirically it corrects the issue, I'm just trying to understand what's happening as a whole.

Reply:
I used the same value used by the button-keyboard as margin. Since it uses flex, the width of the each component needs to match the others, so that they become aligned. At least, this was how I perceived it.